datafusion/expr/src/window_function.rs

```diff
@@ -246,10 +276,10 @@ mod tests {
     #[test]
     fn test_count_return_type() -> Result<()> {
         let fun = find_df_window_func("count").unwrap();
-        let observed = return_type(&fun, &[DataType::Utf8])?;
+        let observed = fun.return_type(&[DataType::Utf8])?;
```

Comment:
This basically shows the new API -- use a method on the `fun` rather than have to find a free function